compose/reset_db.sh

```diff
@@ -0,0 +1,6 @@
+docker-compose stop db
+docker-compose rm -f db
+docker-compose up -d db
+sleep 15
+docker-compose exec -t web npx prisma db push #command to migrate using prisma
```

Comment:
You will need to rebuild your web docker image before it will be able to use the Prisma schema

Reply:
docker-compose rm -f web

Reply:
docker-compose up -d --parallel

Reply:
Added rebuild of the web container